composer.json

```diff
@@ -63,6 +63,7 @@
     "oat-sa/oatbox-extension-installer": "~1.1||dev-master",
     "oat-sa/generis" : ">=15.22",
     "oat-sa/tao-core" : ">=50.24.6",
-    "oat-sa/extension-tao-outcome" : ">=13.0.0"
+    "oat-sa/extension-tao-outcome" : ">=13.0.0",
+    "oat-sa/extension-tao-delivery": ">=15.13.2"
```

Comment:
- Wont it cause cyclic dependency issue and crash the application? 

If you need it only for testing, maybe use in the require-dev section?

Reply:
Let me try 🙂

Reply:
Well, seems here is used only in require-dev https://github.com/oat-sa/extension-tao-outcome/blob/master/composer.json#L75 

Maybe it will not break tao-community 🤞